operator/cmd/mesh/manifest.go

```diff
@@ -36,6 +36,7 @@ func ManifestCmd(logOpts *log.Options) *cobra.Command {
 	mgc := manifestGenerateCmd(args, mgcArgs, logOpts)
 	mdc := manifestDiffCmd(args, mdcArgs)
 	ic := InstallCmd(logOpts)
+	ic.Short += " " + "[Deprecated, use \"istioctl install\". it will be removed in Istio 1.9]"
```

Comment:
Instead of adding `Deprecation` warning. You can move `ic := InstallCmd(logOpts)` to the `istioctl/cmd/root.go`. In that way, it will directly register with `istioctl` main command.